products/compute/api.yaml

```diff
@@ -4209,9 +4209,6 @@ objects:
         name: 'enableFlowLogs'
         description: |
           Whether to enable flow logging for this subnetwork.
-        update_verb: :PATCH
-        update_url:  projects/{{project}}/regions/{{region}}/subnetworks/{{name}}
-        send_empty_value: true
```

Comment:
Agreed that the subnetworks part of this looks weird. Should it be `send_empty_value: true` or is that also part of this copy-and-paste error?